en/components/dialog.md

```diff
@@ -126,6 +126,37 @@ Also we can add two groups of label and input decorated with the [**igxLabel**](
 </div>
 <div class="divider--half"></div>
 
+#### Customized Title and Actions
+
+Dialog title area can be customized using `igxDialogTitle` directive or `igx-dialog-title` selector. The actions area can be customized using `igxDialogActions` directive or `igx-dialog-actions` selector.
+
+```html
+<!-- dialog.component.html -->
+
+<igx-dialog #dialog [closeOnOutsideSelect]="true" message="This will create a new social media account.">
+    <igx-dialog-title>
+        <div class="dialog-container">
+            <igx-icon>account_box</igx-icon>
+            <div class="dialog-title">Create a new account?</div>
+        </div>
+    </igx-dialog-title>
+    <igx-dialog-actions>
```

Comment:
Could you please change this element to `div` with `igxDialogActions` directive applied to demonstrate both approaches.